autogtp/.gitignore

```diff
@@ -2,4 +2,9 @@
 /.qmake.stash
 /autogtp
 /*.sgf
-/*.gz
\ No newline at end of file
+/*.gz
+/moc_*.cpp
+/moc_*.h
+
+# Weight files
+/[0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f]*[0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f]
```

Comment:
technically I should repeat the pattern 64 times but I think if the file starts with 5 hex characters and ends with 5 more it's probably a weight file

Reply:
You can just do `[0-9a-f]{64}` no?

Reply:
No, sadly .gitignore uses a glob pattern match , instead of a full regex

https://stackoverflow.com/questions/33189437/explain-gitignore-pattern-matching